lib/components/viewers/RouteRow.js

```diff
@@ -0,0 +1,160 @@
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+import { Label, Button } from 'react-bootstrap'
+import { VelocityTransitionGroup } from 'velocity-react'
+
+import { getModeFromRoute, getColorAndNameFromRoute } from '../../util/viewer'
```

Comment:
Sort imports.

Reply:
Sorted manually in 9a7139548d78db23db07dfddf9940bd85078072b